src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_COEFFICIENT = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resizing();
+        int bucket = getIndex(key);
+        Node<K, V> node = table[bucket];
+        Node<K, V> newNode = new Node<>(key, value, null);
+        if (node == null) {
+            table[bucket] = newNode;
+        }
+        while (node != null) {
+            if (Objects.equals(key, node.key)) {
+                node.value = value;
+                return;
+            } else if (node.next == null) {
+                node.next = newNode;
+                break;
+            }
+            node = node.next;
+        }
+        size++;
     }
 
     @Override
     public V getValue(K key) {
+        int bucket = getIndex(key);
+        Node<K, V> node = table[bucket];
+        while (node != null) {
+            if (Objects.equals(key, node.key)) {
+                return node.value;
+            }
+            node = node.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void resizing() {
```

Comment:
```suggestion
    private void resize() {
```
method names should start with verb